src/gui/tray/UserLine.qml

```diff
@@ -14,236 +14,237 @@ MenuItem {
     Accessible.role: Accessible.MenuItem
     Accessible.name: qsTr("Account entry")
 
-        RowLayout {
-            id: userLineLayout
-            spacing: 0
-            width: Style.currentAccountButtonWidth
-            height: parent.height
-
-            Button {
-                id: accountButton
-                Layout.preferredWidth: (userLineLayout.width * (5/6))
-                Layout.preferredHeight: (userLineLayout.height)
-                display: AbstractButton.IconOnly
+    RowLayout {
+        id: userLineLayout
+        spacing: 0
+        anchors.fill: parent
+
+        Button {
+            id: accountButton
+            Layout.preferredWidth: (userLineLayout.width * (5/6))
+            Layout.preferredHeight: (userLineLayout.height)
+            display: AbstractButton.IconOnly
+            hoverEnabled: true
+            flat: true
+
+            Accessible.role: Accessible.Button
+            Accessible.name: qsTr("Switch to account") + " " + name
+
+            MouseArea {
+                anchors.fill: parent
                 hoverEnabled: true
-                flat: true
+                onContainsMouseChanged: {
+                    accountStatusIndicatorBackground.color = (containsMouse ? "#f6f6f6" : "white")

```

Comment:
Not needed, but because you are changing code here: Maybe you could abstract this magic numbers into constants.